docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-product-offer-prices-feature-integration.md

```diff
@@ -92,8 +92,6 @@ Make sure that the following changes were applied in transfer objects:
 | PriceProductOfferCriteria | class | created | src/Generated/Shared/Transfer/PriceProductOfferCriteriaTransfer |
 | DataImporterReaderConfiguration | class | created | src/Generated/Shared/Transfer/DataImporterReaderConfigurationTransfer |
 | Currency.Code | attribute | created | src/Generated/Shared/Transfer/CurrencyTransfer |
-| PriceProductFilter.Filter | attribute | created | src/Generated/Shared/Transfer/PriceProductFilterTransfer |
```

Comment:
Do we have to delete them? Propose just to fix attribute names, cos we're registering 
 `PriceProductFilter.productOfferReference` and `PriceProductFilter.quantity` transfer properties in PriceProductOffer and PriceProductOfferVolume accordingly

Reply:
Please add PriceProductFilter.productOfferReference (PriceProductFilter.quantity exists in Prices feature)